docs/services/aws-services-guide.md

```diff
@@ -1,8 +1,5 @@
 ---
-title: Using AWS Services
-toc: true
-weight: 430
-indent: true
+title: Using AWS Services toc: true weight: 430 indent: true
```

Comment:
Looks like this could be the product of a rogue reformat?

Reply:
https://github.com/hasheddan/crossplane/pull/3